lms/static/js/components/tabbed/views/tabbed_view.js

```diff
@@ -44,8 +44,7 @@
                        this.$('a.is-active').removeClass('is-active').attr('aria-selected', 'false');
                        this.$('a[data-index='+index+']').addClass('is-active').attr('aria-selected', 'true');
                        var view = this.tabs[index].view;
-                       view.render();
-                       this.$('.page-content-main').html(view.$el.html());
+                       view.setElement(this.$('.page-content-main')).render();
```

Comment:
Question: Should this generic component really have knowledge of `'.page-content-main'`?